src/components/menu/menu.tsx

```diff
@@ -0,0 +1,42 @@
+import classNames from 'classnames';
+import { useState } from 'react';
+import styles from './menu.module.scss';
+
+const menu = ['all', 'documentary', 'comedy', 'horror', 'crime'];
```

Comment:
Well, I would prefer to use enum for specifying genres to make sure you have a single entry point in your app with values